cogni-libs/src/main/java/zone/cogni/libs/sparqlservice/impl/FusekiConfig.java

```diff
@@ -0,0 +1,65 @@
+package zone.cogni.libs.sparqlservice.impl;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class FusekiConfig extends Config {
+
+  private String updateUrl;
+  private String queryUrl;
+  private String graphStoreUrl;
```

Comment:
are those 3 urls so different and unpredictable? can they not be based on the same "root" url?

Reply:
weird. ok then 👍